### ISSUE TITLE
[IMPROVEMENT] Add volume metric that tracks actual filesystem usage?

### ISSUE THREAD

**Issue by davidfrickert** [longhorn/longhorn#8064]
## Is your improvement request related to a feature? Please describe (👍 if you like this request)

Volume metrics currently show the "actual size" of the volume, but this is can go from relevant to not relevant very quickly, since this value is actually the sum of the disk space that the snapshots are using, and not really the filesystem usage of the volume, so it can easily be bigger than the volume size.
This make this metric useless for the purpose of tracking how close the volume filesystem to allocated size.

## Describe the solution you'd like

Provide a metric that represents the actual filesystem usage of the volume.

## Additional context

Not sure how easily achievable this is, but it can probably only be collected when the volume is attached to a node and mounted on a container?

**Comment by derekbit** [comment]
One idea is to periodically stat the mount point of an attached volume. The method is infeasible for non-attached volume.
@davidfrickert Do you think this enough for monitoring?

**Comment by davidfrickert** [comment]
Yeah I think that is enough, as it should now allow me and other users to create some alarms based on the actual filesystem usage % !

**Comment by joshimoo** [comment]
@derekbit @davidfrickert I think they are already available while a volume is in use via https://github.com/longhorn/longhorn/issues/1821#issue-708899678

**Comment by derekbit** [comment]
@joshimoo Wow, cool! I forgot Kubernetes already monitors the values. Thanks for the hints.

**Comment by davidfrickert** [comment]
> @derekbit @davidfrickert I think they are already available while a volume is in use via https://github.com/longhorn/longhorn/issues/1821#issue-708899678

Ah amazing. Will test it out!
